tests/bga.test.ts

```diff
@@ -35,10 +35,33 @@ test("bga64_w10_h10_grid8x8_p1.27mm", () => {
     .missing("center")
     .p(1.27)
     .soup()
-  // 16pins, 4mm x 4mm, 8x8 grid, 1.27mm pitch
   const svgContent = convertCircuitJsonToPcbSvg(soup)
   expect(svgContent).toMatchSvgSnapshot(
     import.meta.path,
     "bga64_w10_h10_grid8x8_p1.27mm",
   )
 })
+
+test("bga footprint with top-left origin (default)", () => {
+  const soup = fp().bga(8).grid("3x3").p(1).soup()
```

Comment:
```suggestion
  const circuitJsono = fp().bga(8).grid("3x3").p(1).soup()
```

Try to use the variable name `circuitJson` inplace of `soup`

Reply:
in every file soup is used why i need to use circuitJson in place of soup

Reply:
@imrishabh18  review now i replaced soup from circuitJson